meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -546,6 +541,7 @@ OSTREE_KERNEL_ARGS:tegra194 ?= "\${cbootargs} ${OSTREE_KERNEL_ARGS_COMMON}"
 OSTREE_BOOTLOADER:tegra194 = "syslinux"
 
 # Allwinner (sun8i)
+KERNEL_BUILTIN_WIREGUARD:sun8i = "0"
```

Comment:
I think sun8i is using a newer kernel as well:
```
conf/machine/include/sunxi.inc:PREFERRED_PROVIDER_virtual/kernel ?= "linux-mainline"
conf/machine/include/sunxi.inc:PREFERRED_VERSION_linux-mainline ?= "5.15.%"
```

Reply:
In fact, it was the only one I didn't test. but I agree that the out of tree module is not needed in this kernel